llvm/tools/llvm-profgen/PerfReader.cpp

```diff
@@ -670,14 +703,37 @@ bool PerfScriptReader::extractLBRStack(TraceStream &TraceIt,
 
   // Skip the leading instruction pointer.
   size_t Index = 0;
+  // Default to PID 0 if not provided
+  int32_t PID = 0;
```

Comment:
There are multiple places uses 0 as the default PID, maybe we can use a const for this?